### ISSUE TITLE
Update design to use photos and videos from our chapters

### ISSUE THREAD

**Issue by juliaelman** [girldevelopit/gdi-new-site#122]
Currently, we're using a generic photo at the top of the homepage. @Cara-Jo made the suggestion of using photos from our chapters instead (YES!). See screenshot below of current homepage image FYR. 
![screen shot 2015-01-30 at 3 28 19 pm](https://cloud.githubusercontent.com/assets/955558/5982771/a6471976-a894-11e4-8af2-0e51bc6cb57b.png)

**Comment by kstack7** [comment]
Will this be driven by our site on its own (e.g. uploads in admin) or maybe an API from an image-storing site? Would be nice to automate/simplify the image upload process while dynamically pulling new photos.

**Comment by juliaelman** [comment]
@kstack7 all great suggestions! What do you think would work best?

@Cara-Jo is there a nice stock pile of GDI images located somewhere?

**Comment by kstack7** [comment]
I guess it depends on the comfort level of the contributors. Built-in is always a great choice, but not sure if putting it in the back-end of the site is intuitive enough? Or if it can work alongside AA? That'd rock. Is space an issue?

APIs are tempting, but always leave room for lag/disconnection.

We'd definitely need to have some sort of approval process so you don't have a crap pic of a chair as the header should these pull dynamically.

Hand-picking them is probably ideal for the sake of the launch. We can always whip up some crazy integrated masterpiece when the deadlines are outta the way!

**Comment by juliaelman** [comment]
@kstack7 right now we only have one spot where images, other than the chapter leader profile images, are being used: the homepage. I vote that we keep this in mind for future integration, but hold off until we have enough spots,with custom images, to worry about it. Make sense? Sound ok?

**Comment by kstack7** [comment]
Oh for sure...just spitballing beyond the designs to see what it could be. I'd be happy to sniff out some options for whenever we are ready to look at it. Def not a pre-launch project!

**Comment by Cara-Jo** [comment]
I like the idea of having 'hero' images uploadable. It would be great if users could upload an image and then be presented with a way to crop images to the right aspect ratio. I've worked with companies before with uploading images that need to be a specific aspect ratio and found it was helpful for them to have an option to crop on upload. 

@juliaelman I don't have a stock pile of GDI images, but I will send a message out to the leaders asking them to send them our way!

**Comment by juliaelman** [comment]
@Cara-Jo that is a GREAT idea and love the idea of developing this for future releases. As I mentioned to @kstack7, I think it would be good to keep it simple for now :+1: 

Is there a particular newer hero image you'd like to put on the homepage? I think we should just change that in the code and call it a day for now. We're close to release and want to make sure to get this accomplished prior to Galentines Day.

**Comment by Cara-Jo** [comment]
Oh, totally down for doing it simple and dirty. 

I'll check my photo archives for some awesome student photos.

**Comment by Cara-Jo** [comment]
I'm going to create an issue with the future implementation of a hero image upload in the admin. That way we can bring the focus of this issue back to finding the most perfect image for the home page.

**Comment by juliaelman** [comment]
@Cara-Jo :+1:

**Comment by juliaelman** [comment]
@Cara-Jo okay to assign this one to you?

**Comment by Cara-Jo** [comment]
Sure can!!! Let's get it done!

**Comment by juliaelman** [comment]
:rainbow: @Cara-Jo

**Comment by corinnepw** [comment]
Could we use this one in the meantime? I cropped to 1400 x 499.
![leaders](https://cloud.githubusercontent.com/assets/4427638/6542673/fb883f6e-c4cd-11e4-92a2-f9ca6b83390d.png)
+1 having a pool of 5-10 images to have a different one appear on each refresh of the page.

Also @Cara-Jo I took some really solid images of leaders building the site at the summit w your fancy :camera:. The lighting at Meetup HQ was lovely, and I think we might have a good one or two from there. Are they shared somewhere? On Dropbox?

**Comment by kstack7** [comment]
Heyo @corinnepw ...my immediate thought is that the animated text will have to be changed to a lighter color. I do love the idea of using our group photo so maybe a designer would like to play with what looks best?

**Comment by Cara-Jo** [comment]
I’ve tried this image in the past on the home page and it is way too busy for it. It’s better suited for an inside page where there isn’t much going on. 

-- 
CARA JO MILLER | Co-founder | Girl Develop It Boulder/Denver | 248.230.0910 | carajo@girldevelopit.com | girldevelopit.com | @GDIboulder | @GDIDenver | @corrinajo

On March 8, 2015 at 11:08:55 PM, kstack7 (notifications@github.com) wrote:

Heyo @corinnepw ...my immediate thought is that the animated text will have to be changed to a lighter color. I do love the idea of using our group photo so maybe a designer would like to play with what looks best?

**Comment by kstack7** [comment]
Yeah...I thought maybe we had discussed this one before. We'll find one!

**Comment by corinnepw** [comment]
@Cara-Jo @kstack7 noted! Can we find a way to get this on the Donate page?

**Comment by corinnepw** [comment]
Scratch that — this would be best on **About**!

**Comment by mmcelaney** [comment]
I just want to drop my two cents and say that we should make a concerted effort to represent people who are not white on the homepage. Unfortunately, Vermont is the pastiest place on earth so I can't really contribute diverse imagery from my chapter, but I'm hoping we have chapters that can help us with this?

**Comment by therufs** [comment]
+1000000

**Comment by corinnepw** [comment]
@mmcelaney Absolutely agree— we should always strive to show images that are representative of our incredibly diverse communities. We should look for images that show women of color and age diversity.

**Comment by kstack7** [comment]
I think we should reconsider the text layout/design as well since it tends to limit the images in use. I know we tried a few other options, but they only worked well with consistently lighter photo options. Let's explore that more, too :)

**Comment by NewtCobell** [comment]
I'd just like to chime in and mention that WOCinTechChat has _amazing_ stock photos of actual women of color who actually work in tech, and they're free to use with attribution! [https://www.flickr.com/photos/wocintechchat/albums](https://www.flickr.com/photos/wocintechchat/albums)
